content/docs/concepts/traffic-management/traffic-routing/index.md

```diff
@@ -0,0 +1,135 @@
+> to resolve the FQDN using the DNS service present in their platform of
+> choice: `kube-dns`, `mesos-dns`, etc.
+
+## Canary rollout {#canary}
+
+Use a canary rollout to introduce a new version of a service while testing it
+using a small percentage of user traffic. If the test is successful, you can
+gradually increase the percentage until all the traffic is moved to the new
+version. If anything goes wrong along the way, you can abort the rollout and
+return the traffic to the old version.
+
+Although container orchestration platforms like Docker, Mesos, Marathon, or
+Kubernetes provide features that support canary rollout, they use instance
+scaling to manage the traffic distribution, which limits them.
+
+For example, to send 10% of traffic to a canary version requires 9 instances of
+the old version to be running for every 1 instance of the canary. This becomes
+particularly difficult in production deployments where autoscaling is needed.
+When traffic load increases, the autoscaler needs to scale instances of both
+versions concurrently, making sure to keep the instance ratio the same.
+
+Additionally, the instance deployment approach only supports a simple, random
+percentage, canary rollout. You can't limit the visibility of the canary to
+requests based on some specific criteria.
+
+With Istio, traffic routing and instance deployment are completely independent
+functions. The number of instances implementing the services can scale up and
+down based on traffic load completely orthogonal to the version traffic
+routing. This model makes managing a canary version in the presence of
+autoscaling a much simpler problem. To learn more about canary deployments
+and the interoperability of canary deployments and autoscaling when using
+Istio visit our blog post on [Canary Deployments](/blog/2017/0.1-canary/).
+
+## Splitting traffic between versions {#splitting}
+
+Each route rule identifies one or more weighted workloads to call when the rule
+is activated. Each workload corresponds to a specific version of the
+destination service. You use subsets to express the versions. If there are
+multiple registered instances with the specified subset, **Pilot** routes them
+based on the load balancing policy configured for the service or using
+round-robin by default.
+
+For example, the following rule routes 25% of traffic for the `reviews` service
+to instances with the `v2` subset and the remaining 75% of traffic to `v1`:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: reviews
+spec:
+  hosts:
+    - reviews
+  http:
+  - route:
+    - destination:
+        host: reviews
+        subset: v1
+      weight: 75
+    - destination:
+        host: reviews
+        subset: v2
+      weight: 25
+{{< /text >}}
+
+To learn more about configuring destination rules, visit our [Destination Rules concept](../destination-rules).
```

Comment:
Shouldn't we be sending them to the virtual services page, not the destination rules page?